soc/renesas/ra/ra8m1/soc.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2024 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file SoC configuration macros for the Renesas RA8 family MCU
```

Comment:
nitpick: `RA8` -> `RA8M1`

Reply:
We update it. Thank you for your feedback.